Wikipedia/Code/WMFColumnarCollectionViewLayout.m

```diff
@@ -103,6 +103,14 @@ - (void)prepareLayout {
     [super prepareLayout];
 }
 
+- (CGPoint)targetContentOffsetForProposedContentOffset:(CGPoint)proposedContentOffset {
```

Comment:
In AdvancedCollectionView they implemented the withScrollingVelocity version without modifying it so I figured they should be implemented. Seemed OK either way, but we'll eventually need to modify the values.